client/src/components/right/results/entry/index.js

```diff
@@ -1,66 +0,0 @@
-import React from 'react';
```

Comment:
@irbab00n It still says here that this file is being deleted in its entirety. Shouldn't it be the opposite?